### ISSUE TITLE
Crash when sorting with "Date Played"

### ISSUE THREAD

**Issue by faustaahmad** [fast4x/RiMusic#3208]
### Steps to reproduce the bug

**・Settings → Data → Save to Backup** (if you wanted to reproduce)
・Songs
・Hold  `↓`  button
・Select "Date Played"
・crash + wont be able to open the app again till clear data/reinstall (crash)

Unlike previous issue (different favorite button), user still able to play the song via notification window

### Expected behavior

Songs will be sorted by Date Played
no crash

### Actual behavior

crash

### RiMusic version

0.6.47.1 release

### Android version

Android 11

**Comment by aneesh1122** [comment]
The app crashes only if I sort by date played and then scroll down.

Anyways here's the crash log

2024-08-11T00:09:36.960:	java.lang.IllegalArgumentException: Key "7Cc4UsvNFBU" was already used. If you are using LazyColumn/Row please make sure you provide a unique key for each item.
	 G0.C.H(SourceFile:151) 
	 E.T.a(SourceFile:34) 
	 C.m.b(SourceFile:21) 
	 d8.d.l0(SourceFile:25) 
	 C.n.g(SourceFile:592) 
	 A6.Y.g(SourceFile:758) 
	 G0.E.g(SourceFile:74) 
	 I0.u.a(SourceFile:38) 
	 q0.Y.i(SourceFile:1) 
	 I0.A.a(SourceFile:15) 
	 B0.b.a(SourceFile:363) 
	 e1.c.y(SourceFile:114) 
	 h0.t.a(SourceFile:52) 
	 h0.u.c(SourceFile:140) 
	 I0.m0.a(SourceFile:3) 
	 I0.K.s0(SourceFile:133) 
	 I0.F.N(SourceFile:17) 
	 I0.T.c(SourceFile:3) 
	 I0.T.k(SourceFile:102) 
	 J0.w.x(SourceFile:8) 
	 I0.F.k(SourceFile:41) 
	 C.x.b(SourceFile:1757) 
	 y.r.a(SourceFile:17) 
	 y.K0.a(SourceFile:65) 
	 w.n.b(SourceFile:252) 
	 y.b.b(SourceFile:115) 
	 y.U.v(SourceFile:59) 
	 d7.a.k(SourceFile:9) 
	 w7.J.run(SourceFile:113) 
	 J0.b0.h0(SourceFile:7) 
	 J0.a0.run(SourceFile:3) 
	 android.os.Handler.handleCallback(Handler.java:942) 
	 android.os.Handler.dispatchMessage(Handler.java:99) 
	 android.os.Looper.loopOnce(Looper.java:211) 
	 android.os.Looper.loop(Looper.java:300) 
	 android.app.ActivityThread.main(ActivityThread.java:8503) 
	 java.lang.reflect.Method.invoke(Native Method) 
	 com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561) 
	 com.android.internal.os.ZygoteInit.main(ZygoteInit.java:954)

**Comment by faustaahmad** [comment]
Even without scrolling, it still crash on me with this backup
https://gofile.io/d/qVxlxG

**Comment by twistios** [comment]
It probably is the same issue as I had. So it should be fixed in the newest custom version.

**Comment by aneesh1122** [comment]
> It probably is the same issue as I had. So it should be fixed in the newest custom version.

I'm using the newest custom version

**Comment by twistios** [comment]
There where multiple custom versions today.

**Comment by twistios** [comment]
I sounds like my issue because it seems like the same crash log and I had also sorting by date played enabled.

**Comment by fast4x** [comment]
You have a multiple with same keys, I'll improve it

**Comment by twistios** [comment]
I am again able to reproduce this error with my debug version when I import the backup from my main app. It is a different key then any stack traces I have seen about this issue before.

<details><summary>Stack trace</summary>
<p>

````
23:54:01.746  E  FATAL EXCEPTION: main
                 Process: it.fast4x.rimusic.debug, PID: 16069
                 java.lang.IllegalArgumentException: Key "xopAcWEKrto" was already used. If you are using LazyColumn/Row please make sure you provide a unique key for each item.
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState.subcompose(SubcomposeLayout.kt:453)
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState$Scope.subcompose(SubcomposeLayout.kt:895)
                 	at androidx.compose.foundation.lazy.layout.LazyLayoutMeasureScopeImpl.measure-0kLqBqw(LazyLayoutMeasureScope.kt:125)
                 	at androidx.compose.foundation.lazy.LazyListMeasuredItemProvider.getAndMeasure-0kLqBqw(LazyListMeasuredItemProvider.kt:55)
                 	at androidx.compose.foundation.lazy.LazyListMeasuredItemProvider.getAndMeasure-0kLqBqw$default(LazyListMeasuredItemProvider.kt:49)
                 	at androidx.compose.foundation.lazy.LazyListMeasureKt.measureLazyList-x0Ok8Vo(LazyListMeasure.kt:214)
                 	at androidx.compose.foundation.lazy.LazyListKt$rememberLazyListMeasurePolicy$1$1.invoke-0kLqBqw(LazyList.kt:325)
                 	at androidx.compose.foundation.lazy.LazyListKt$rememberLazyListMeasurePolicy$1$1.invoke(LazyList.kt:183)
                 	at androidx.compose.foundation.lazy.layout.LazyLayoutKt$LazyLayout$3$2$1.invoke-0kLqBqw(LazyLayout.kt:119)
                 	at androidx.compose.foundation.lazy.layout.LazyLayoutKt$LazyLayout$3$2$1.invoke(LazyLayout.kt:112)
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState$createMeasurePolicy$1.measure-3p2s80s(SubcomposeLayout.kt:725)
                 	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:135)
                 	at androidx.compose.ui.graphics.SimpleGraphicsLayerModifier.measure-3p2s80s(GraphicsLayerModifier.kt:646)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:316)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:315)
                 	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:503)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:502)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:258)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:133)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeMeasureSnapshotReads$ui_release(OwnerSnapshotObserver.kt:113)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1771)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.access$performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:40)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:692)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.measure-BRTryo0(LayoutNodeLayoutDelegate.kt:668)
                 	at androidx.compose.foundation.layout.BoxMeasurePolicy.measure-3p2s80s(Box.kt:173)
                 	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:135)
                 	at androidx.compose.foundation.layout.FillNode.measure-3p2s80s(Size.kt:699)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.foundation.layout.FillNode.measure-3p2s80s(Size.kt:699)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:316)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:315)
                 	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:503)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:502)
23:54:01.751  E  	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:258)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:133)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeMeasureSnapshotReads$ui_release(OwnerSnapshotObserver.kt:113)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1771)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.access$performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:40)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:692)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.measure-BRTryo0(LayoutNodeLayoutDelegate.kt:668)
                 	at androidx.compose.animation.AnimatedEnterExitMeasurePolicy.measure-3p2s80s(AnimatedVisibility.kt:812)
                 	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:135)
                 	at androidx.compose.animation.EnterExitTransitionModifierNode.measure-3p2s80s(EnterExitTransition.kt:1173)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.ui.graphics.BlockGraphicsLayerModifier.measure-3p2s80s(GraphicsLayerModifier.kt:578)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.animation.AnimatedContentKt$AnimatedContent$6$1$1$1.invoke-3p2s80s(AnimatedContent.kt:781)
                 	at androidx.compose.animation.AnimatedContentKt$AnimatedContent$6$1$1$1.invoke(AnimatedContent.kt:780)
                 	at androidx.compose.ui.layout.LayoutModifierImpl.measure-3p2s80s(LayoutModifier.kt:294)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:316)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:315)
                 	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:503)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:502)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:258)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:133)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeMeasureSnapshotReads$ui_release(OwnerSnapshotObserver.kt:113)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1771)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.access$performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:40)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:692)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.measure-BRTryo0(LayoutNodeLayoutDelegate.kt:668)
                 	at androidx.compose.animation.AnimatedContentMeasurePolicy.measure-3p2s80s(AnimatedContent.kt:837)
                 	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:135)
                 	at androidx.compose.foundation.layout.FillNode.measure-3p2s80s(Size.kt:699)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:316)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:315)
                 	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:503)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:502)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:258)
23:54:01.754  E  	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:133)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeMeasureSnapshotReads$ui_release(OwnerSnapshotObserver.kt:113)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1771)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.access$performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:40)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:692)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.measure-BRTryo0(LayoutNodeLayoutDelegate.kt:668)
                 	at androidx.compose.foundation.layout.RowColumnMeasurePolicyKt.measure(RowColumnMeasurePolicy.kt:119)
                 	at androidx.compose.foundation.layout.RowColumnMeasurePolicyKt.measure$default(RowColumnMeasurePolicy.kt:74)
                 	at androidx.compose.foundation.layout.RowMeasurePolicy.measure-3p2s80s(Row.kt:145)
                 	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:135)
                 	at androidx.compose.foundation.layout.FillNode.measure-3p2s80s(Size.kt:699)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:316)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:315)
                 	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:503)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:502)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:258)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:133)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeMeasureSnapshotReads$ui_release(OwnerSnapshotObserver.kt:113)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1771)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.access$performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:40)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:692)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.measure-BRTryo0(LayoutNodeLayoutDelegate.kt:668)
                 	at androidx.compose.foundation.layout.BoxMeasurePolicy.measure-3p2s80s(Box.kt:173)
                 	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:135)
                 	at androidx.compose.foundation.layout.FillNode.measure-3p2s80s(Size.kt:699)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.foundation.layout.PaddingValuesModifier.measure-3p2s80s(Padding.kt:472)
                 	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(LayoutModifierNodeCoordinator.kt:188)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:316)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:315)
                 	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:503)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:502)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:258)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:133)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeMeasureSnapshotReads$ui_release(OwnerSnapshotObserver.kt:113)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1771)
23:54:01.757  E  	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.access$performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:40)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:692)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.measure-BRTryo0(LayoutNodeLayoutDelegate.kt:668)
                 	at androidx.compose.material3.ScaffoldKt$ScaffoldLayout$1$1.invoke-0kLqBqw(Scaffold.kt:263)
                 	at androidx.compose.material3.ScaffoldKt$ScaffoldLayout$1$1.invoke(Scaffold.kt:140)
                 	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState$createMeasurePolicy$1.measure-3p2s80s(SubcomposeLayout.kt:725)
                 	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(InnerNodeCoordinator.kt:135)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:316)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$performMeasureBlock$1.invoke(LayoutNodeLayoutDelegate.kt:315)
                 	at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:2441)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:502)
                 	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:258)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:133)
                 	at androidx.compose.ui.node.OwnerSnapshotObserver.observeMeasureSnapshotReads$ui_release(OwnerSnapshotObserver.kt:113)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:1771)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate.access$performMeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:40)
                 	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(LayoutNodeLayoutDelegate.kt:692)
                 	at androidx.compose.ui.node.LayoutNode.remeasure-_Sx5XlM$ui_release(LayoutNode.kt:1221)
                 	at androidx.compose.ui.node.LayoutNode.remeasure-_Sx5XlM$ui_release$default(LayoutNode.kt:1212)
                 	at androidx.compose.ui.node.MeasureAndLayoutDelegate.doRemeasure-sdFAvZA(MeasureAndLayoutDelegate.kt:369)
                 	at androidx.compose.ui.node.MeasureAndLayoutDelegate.remeasureAndRelayoutIfNeeded(MeasureAndLayoutDelegate.kt:566)
                 	at androidx.compose.ui.node.MeasureAndLayoutDelegate.remeasureAndRelayoutIfNeeded$default(MeasureAndLayoutDelegate.kt:534)
                 	at androidx.compose.ui.node.MeasureAndLayoutDelegate.measureAndLayout(MeasureAndLayoutDelegate.kt:390)
                 	at androidx.compose.ui.platform.AndroidComposeView.measureAndLayout(AndroidComposeView.android.kt:1258)
                 	at androidx.compose.ui.node.Owner.measureAndLayout$default(Owner.kt:242)
                 	at androidx.compose.ui.platform.AndroidComposeView.dispatchDraw(AndroidComposeView.android.kt:1527)
                 	at android.view.View.draw(View.java:25180)
                 	at android.view.View.updateDisplayListIfDirty(View.java:24036)
                 	at android.view.View.draw(View.java:24907)
                 	at android.view.ViewGroup.drawChild(ViewGroup.java:4780)
                 	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4538)
                 	at android.view.View.updateDisplayListIfDirty(View.java:24022)
                 	at android.view.View.draw(View.java:24907)
                 	at android.view.ViewGroup.drawChild(ViewGroup.java:4780)
                 	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4538)
                 	at android.view.View.updateDisplayListIfDirty(View.java:24022)
                 	at android.view.View.draw(View.java:24907)
                 	at android.view.ViewGroup.drawChild(ViewGroup.java:4780)
                 	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4538)
                 	at android.view.View.updateDisplayListIfDirty(View.java:24022)
                 	at android.view.View.draw(View.java:24907)
                 	at android.view.ViewGroup.drawChild(ViewGroup.java:4780)
                 	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4538)
                 	at android.view.View.updateDisplayListIfDirty(View.java:24022)
                 	at android.view.View.draw(View.java:24907)
                 	at android.view.ViewGroup.drawChild(ViewGroup.java:4780)
                 	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4538)
23:54:01.758  E  	at android.view.View.updateDisplayListIfDirty(View.java:24022)
                 	at android.view.View.draw(View.java:24907)
                 	at android.view.ViewGroup.drawChild(ViewGroup.java:4780)
                 	at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4538)
                 	at com.android.internal.policy.DecorView.dispatchDraw(DecorView.java:1179)
                 	at android.view.View.draw(View.java:25180)
                 	at com.android.internal.policy.DecorView.draw(DecorView.java:1161)
                 	at android.view.View.updateDisplayListIfDirty(View.java:24036)
                 	at android.view.ThreadedRenderer.updateViewTreeDisplayList(ThreadedRenderer.java:768)
                 	at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:774)
                 	at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:872)
                 	at android.view.ViewRootImpl.draw(ViewRootImpl.java:6051)
                 	at android.view.ViewRootImpl.performDraw(ViewRootImpl.java:5706)
                 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:4795)
                 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:3288)
                 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:11344)
                 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1689)
                 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1698)
                 	at android.view.Choreographer.doCallbacks(Choreographer.java:1153)
                 	at android.view.Choreographer.doFrame(Choreographer.java:1079)
                 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1646)
                 	at android.os.Handler.handleCallback(Handler.java:958)
                 	at android.os.Handler.dispatchMessage(Handler.java:99)
                 	at android.os.Looper.loopOnce(Looper.java:230)
                 	at android.os.Looper.loop(Looper.java:319)
                 	at android.app.ActivityThread.main(ActivityThread.java:8919)
                 	at java.lang.reflect.Method.invoke(Native Method)
                 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:578)
                 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
````

</p>
</details>

**Comment by fast4x** [comment]
Fixed, available from version 0.6.48

**Comment by fast4x** [comment]
Download new custom version 
https://github.com/fast4x/RiMusic/tree/master/customVersion
